assets/node-exporter/daemonset.yaml

```diff
@@ -2,43 +2,49 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   labels:
+    app.kubernetes.io/component: exporter
+    app.kubernetes.io/managed-by: cluster-monitoring-operator
     app.kubernetes.io/name: node-exporter
-    app.kubernetes.io/version: v1.0.1
+    app.kubernetes.io/part-of: openshift-monitoring
+    app.kubernetes.io/version: 1.0.1
   name: node-exporter
   namespace: openshift-monitoring
 spec:
   selector:
     matchLabels:
+      app.kubernetes.io/component: exporter
+      app.kubernetes.io/managed-by: cluster-monitoring-operator
       app.kubernetes.io/name: node-exporter
+      app.kubernetes.io/part-of: openshift-monitoring
   template:
     metadata:
       labels:
+        app.kubernetes.io/component: exporter
+        app.kubernetes.io/managed-by: cluster-monitoring-operator
         app.kubernetes.io/name: node-exporter
-        app.kubernetes.io/version: v1.0.1
+        app.kubernetes.io/part-of: openshift-monitoring
+        app.kubernetes.io/version: 1.0.1
     spec:
       containers:
       - args:
         - --web.listen-address=127.0.0.1:9100
-        - --path.procfs=/host/proc
         - --path.sysfs=/host/sys
         - --path.rootfs=/host/root
         - --no-collector.wifi
         - --collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+|var/lib/kubelet/pods/.+)($|/)
+        - --collector.netclass.ignored-devices=^(veth.*)$
```

Comment:
Are we sure we wanted to exclude/ignore this?

Reply:
Yes, this is part of the resiliency epic and removing unnecessary data. Metrics for those interfaces don't give us anything new as the relevant data is also exposed by kubelet and we mainly use kubelet data.

Reply:
This is coming from upstream https://github.com/prometheus-operator/kube-prometheus/pull/890

Reply:
Thanks!